src/us/kbase/userprofile/UserProfileServer.java

```diff
@@ -30,15 +30,12 @@
  */
 public class UserProfileServer extends JsonServerServlet {
     private static final long serialVersionUID = 1L;
-    private static final String version = "0.2.1";
-    private static final String gitUrl = "https://github.com/kbase/user_profile";
-    // the git commmit hash hasn't been updated in 2 years as of 3/1/19 and there doesn't appear
-    // to be a mechanism to update it correctly, as it needs to be SDK compiled from the 
-    // current commit and then built into a server. Any checked in commit must be wrong.
-    private static final String gitCommitHash = "";
+    private static final String version = "0.0.1";
+    private static final String gitUrl = "https://github.com/kbase/user_profile.git";
+    private static final String gitCommitHash = "9c4a73d699c07f6c18845b5eebc50b58eb3fce06";
```

Comment:
where does this git commit come from? `kb-sdk compile`?

Reply:
Right